README.md

```
@@ -52,6 +52,11 @@ Instructions for setting up and running the various scripts locally
 
 ### Setup
 
+**Prerequistes**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)